### PR TITLE
Remove deprecated draft-js hooks usage

### PR DESCRIPTION
**Pull request by TrySound**
All onDownArrow, onUpArrow, onLeftArrow, onRightArrow, onEscape, onTab
usages are replaced with keyBindingFn.

Their mentions are dropped from HOW_TO_CREATE_A_PLUGIN tut.

### CODE REVIEW

File: HOW_TO_CREATE_A_PLUGIN.md [hunk 1]
```diff
@@ -58,12 +58,6 @@ A plugin accepts all standard props the Draft.js Editor Component uses. [See det
 - [handlePastedFiles](https://facebook.github.io/draft-js/docs/api-reference-editor.html#handlepastedfiles)
 - [handleDroppedFiles](https://facebook.github.io/draft-js/docs/api-reference-editor.html#handledroppedfiles)
 - [handleDrop](https://facebook.github.io/draft-js/docs/api-reference-editor.html#handledrop)
-- [onEscape](https://facebook.github.io/draft-js/docs/api-reference-editor.html#onescape)
-- [onTab](https://facebook.github.io/draft-js/docs/api-reference-editor.html#ontab)
-- [onUpArrow](https://facebook.github.io/draft-js/docs/api-reference-editor.html#onuparrow)
-- [onRightArrow](https://draftjs.org/docs/api-reference-editor.html#onrightarrow)
-- [onDownArrow](https://facebook.github.io/draft-js/docs/api-reference-editor.html#ondownarrow)
-- [onLeftArrow](https://draftjs.org/docs/api-reference-editor.html#onleftarrow)
```
Comment (juliankrispel) [inline HOW_TO_CREATE_A_PLUGIN.md]: :+1:

File: draft-js-emoji-plugin/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## to be released
+
+- removed deprecated draft-js hooks (onUpArrow, onDownArrow, onEscape, onTab) usage
+
```
Comment (juliankrispel) [inline draft-js-emoji-plugin/CHANGELOG.md]: yaasss :+1:

**Comment by pederjohnsen** [comment]
@TrySound think this is broken? Using latest version, arrow keys, tab and esc no longer do anything in suggestions lists :thinking:

**Comment by juliankrispel** [comment]
@pederjohnsen can you open an issue and give us like a codesandbox example?

**Comment by pederjohnsen** [comment]
@juliankrispel I'll double check it's not just caused by something in our forks.. :ok_hand:

**Comment by TrySound** [comment]
@pederjohnsen Make sure you don't have keyBIndingFn with getDefaultKeyBinding(event)

**Comment by pederjohnsen** [comment]
@TrySound ah, thanks, that seems to be the reason :ok_hand: 

Does this mean such function can no longer be used? :thinking: 
```
  keyBindingFunction(event) {
    const {submitOnEnterKey, singleLine} = this.props;
    const {hasCommandModifier} = KeyBindingUtil;

    if (submitOnEnterKey && (event.keyCode === 13 || event.keyCode === 10) && (singleLine || !event.shiftKey)) {
      return 'submit';
    }

    if (hasCommandModifier(event) && (event.keyCode === 13 || event.keyCode === 10)) {
      return 'submit';
    }

    return getDefaultKeyBinding(event);
  }
```

**Comment by TrySound** [comment]
The problem is only `return getDefaultKeyBinding(event)` in the end. It should be removed.

**Comment by pederjohnsen** [comment]
@TrySound great :ok_hand: thanks for the clarification :tada:

**Comment by coldshower** [comment]
@TrySound
Hey Bogdan, thanks for the guidance. What do you think about updating the documentation, which recommends adding a `return getDefaultKeyBinding(event)`? 

https://draftjs.org/docs/advanced-topics-key-bindings#example

I can make the PR unless I'm missing something.

**Comment by Stalinko** [comment]
Gosh! I've spent whole day today trying to understand why key bindings for mentions popup broke at some moment. Even re-read the docs and source code, but the issue was in just 1 line! Guys, that should really be addressed in the docs! @coldshower is absolutely right

**Pull request merged**